epoxy-processor/src/main/java/com/airbnb/epoxy/EpoxyProcessor.java

```diff
@@ -458,6 +402,44 @@ private void generateClassForModel(ClassToGenerateInfo info) throws IOException
     return methods;
   }
 
+  /**
+   * Generates default implementations of certain model methods if the model is abstract and doesn't
+   * implement them.
+   */
+  private Iterable<MethodSpec> generateDefaultMethodImplementations(ClassToGenerateInfo info)
```

Comment:
@gpeal I can't think of any good cases where you would want the generated class to be abstract. That doesn't seem useful to me, interested to hear if you know a case.

This isn't creating an empty implementation, it's generating a default implementation. In this case the default implementation is very likely what you would have written manually anyway, so it is helping to reduce boilerplate.